test/battle/move_effect/future_sight.c

```diff
@@ -0,0 +1,133 @@
+#include "global.h"
+#include "test/battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gMovesInfo[MOVE_SEED_FLARE].power == gMovesInfo[MOVE_FUTURE_SIGHT].power);
+}
+
+SINGLE_BATTLE_TEST("Future Sight uses the Attack stat of the original user without modifiers")
```

Comment:
Shouldn't this be Special Attack? In which case we also need to check the category of both moves in the assumptions.

Reply:
yes